scipy/stats/vonmises_cython.pyx

```diff
@@ -30,7 +30,7 @@ cdef double von_mises_cdf_series(double k,double x,unsigned int p):
         return 0.5 + x / (2 * NPY_PI) + V / NPY_PI
 
 cdef von_mises_cdf_normalapprox(k, x):
-    b = sqrt(NPY_2_PI) * np.exp(k) / i0(k)
+    b = sqrt(NPY_2_PI) / i0e(k) # Check for negative k
```

Comment:
Actually the comment can go - you only get to this branch if k>50. But in fact the von Mises distribution does make sense for k<0 (same as that for -k but shifted by pi), and this is not really implemented in scipy.